UnityProject/Assets/Scripts/Health/IElectrocutable.cs

```diff
@@ -0,0 +1,12 @@
+/// <summary>
+/// Implement this on a component to cause that component to do something when electrocuted.
+/// It's up to the component what it does in response to electrocution.
+/// </summary>
+public interface IElectrocutable
+{
+	/// <summary>
+	/// Expose this component to electrocution.
+	/// </summary>
+	/// <param name="exposure">information on the exposure</param>
+	ElectrocutionSeverity Electrocute(Electrocution exposure);
```

Comment:
If the contract is "do whatever it wants", then why have a return value? What is the contract for the return value? IMO it should just be void. The logic relying on the return value should be moved into the implementation or the class that's consuming the instance of IElectrocutable.

For example of a possible issue, I notice that ElectrocutionSeverity is a return value, but it's also a field on the exposure. My implementation could return None but could also modify exposure to have a severity of Mild (which is also why I suggest making Electrocution immutable and removing severity from Electrocution).